### PR TITLE
Dave 9

### PR DESCRIPTION
**Pull request by crystal95**
Prevent pop-ups when hovering s data point . Instead display the coordinate values below the plot .

### CODE REVIEW

File: src/main/js/electron/templates/index.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 {% block body %}
 <head>
 
-<script type="text/javascript" src="{{ url_for('static', filename='plotly-latest.min.js') }}"></script>
+  <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```
Comment (pbalm) [inline src/main/js/electron/templates/index.html]: Is this a mistake? Better to use the local file, no?

File: src/main/js/electron/templates/index.html [hunk 2]
```diff
@@ -165,12 +167,80 @@
...
+
 });
 });
 
 
 </script>
 
+<script> 
+
+</script>
```
Comment (pbalm) [inline src/main/js/electron/templates/index.html]: Empty script?

File: src/main/js/electron/templates/index.html (hunk 1 above)
Comment (crystal95) [inline src/main/js/electron/templates/index.html]: Oh , yes . I was checking something and forgot change !

File: src/main/js/electron/templates/index.html (hunk 2 above)
Comment (crystal95) [inline src/main/js/electron/templates/index.html]: Not needed actually !

**Pull request merged**